docs/includes/reactivestreams/rsoperators.adoc

```diff
@@ -30,13 +42,9 @@ include::{rootdir}/includes/dependencies.adoc[]
 </dependency>
 ----
 
-== Reactive Streams Operators
+== Usage
 
-Implementation of
-link:{microprofile-rs-operators-spec-url}[MicroProfile Reactive Streams Operators]
-specification. A standardised tool for manipulation with https://www.reactive-streams.org/[Reactive Streams],
-provides set of operators as so called stages,
-and the builders to prepare graphs of stages for streams to be build from.
+The specification provides a set of operators as so called stages, and the builders to prepare graphs of stages for streams to be build from.
```

Comment:
The MicroProfile Reactive Streams Operators specification provides a set of operators within stages, as well as the builders to prepare graphs of stages from which steams can be built.